odxtools/compumethods/texttablecompumethod.py

```diff
@@ -36,7 +36,7 @@ def _get_scales(self):
         return scales
 
     def convert_physical_to_internal(self, physical_value):
-        scale: CompuScale = next(
+        scale = next(
```

Comment:
why?

Reply:
mypy started to warn about this:
```
```
> mypy --ignore-missing-imports odxtools examples/ tests/
odxtools/compumethods/texttablecompumethod.py:39: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
Success: no issues found in 150 source files
```

The real fix would be to add type annotations to all `convert_physical_to_internal()` methods, but I was not in the mood to open that can of worms...

Reply:
(note: comprehensive type annotations for everything are currently relatively high on my list of priorities...)

Reply:
please keep it, even if mypy is not checking it, that type hint helps the IDE to provide correct suggestions.

mypy is informing you, it's not a warning or an error.

Reply:
good point. (as a user of emacs without a language server, I do not notice such things...) fixed: 75967cd9cc88baccba3961d2cb8655f8d23fd15f.